frontends/web/src/i18n/i18n.ts

```diff
@@ -79,7 +80,6 @@ i18Init.init({
         enabled: i18nEditorActive,
         autoOpen: true,
         mode: 'iframe', // 'window',
-        projectId: locizeProjectID,
```

Comment:
🤔

Reply:
I haven't tested the special build with the in-app locize editor, had to up the locize npm module typescript now complains about projectId, I am affraid that with this upgrade the in app editor can't work anymore.